react/components/ProfileMetadata/components/ProfileMetadataActions/index.js

```diff
@@ -22,26 +22,27 @@ const UserMessageButton = styled(MessageButton)`
 
 export default class ProfileMetadataActions extends Component {
   static propTypes = {
-    user: propType(profileMetadataActionsFragment).isRequired,
+    identifiable: propType(profileMetadataActionsFragment).isRequired,
   }
 
   render() {
-    const { user } = this.props;
+    const { identifiable } = this.props;
```

Comment:
Do we imagine groups being able to follow as an entity?

Reply:
Not currently no. You can't *embody* a group; you can just be a member.